doc/usermanual/lighttable/lighttable.xml

```diff
@@ -186,18 +186,18 @@
 
     <para>
       In both modes you can navigate left/right/up/down through your collection using
-      &larr;/&rarr;/&uarr;/&darr;. Pressing <emphasis>g</emphasis> in filemanager mode goes to
-      the top, <emphasis>shift-g</emphasis> to the bottom, <emphasis>PageUp</emphasis> brings
-      you one page up and <emphasis>PageDown</emphasis> one page down.
+      &larr;/&rarr;/&uarr;/&darr;. Pressing <emphasis>G</emphasis> in filemanager mode goes to
+      the top, <emphasis>Shift + G</emphasis> to the bottom, <emphasis>Page Up</emphasis> brings
+      you one page up and <emphasis>Page Down</emphasis> one page down.
     </para>
 
     <para>
       You can select the image under the pointer by <emphasis>left-clicking</emphasis> on its
       thumbnail or by pressing <emphasis>return</emphasis>. A range of images is selected by
       <emphasis>clicking</emphasis> on the first one and then
-      <emphasis>shift-clicking</emphasis> on the last one. Images are added or removed from your
-      selection by <emphasis>ctrl-clicking</emphasis> on their thumbnails or pressing
-      <emphasis>space</emphasis>.
+      <emphasis>Shift-clicking</emphasis> on the last one. Images are added or removed from your
+      selection by <emphasis>Ctrl-clicking</emphasis> on their thumbnails or pressing
+      <emphasis>space bar</emphasis>.
```

Comment:
The key is called `space` and not `space bar`. That's as if you added `key` to all other keys.